app/helpers/ems_container_helper/textual_summary.rb

```diff
@@ -18,7 +18,25 @@ def textual_group_relationships
   end
 
   def textual_group_status
-    %i(refresh_status)
+    textual_authentications + %i(refresh_status)
+  end
+
+  def textual_authentications
+    authentications = @ems.authentications
+    return [{:label => _("Default Authentication"), :title => t = _("None"), :value => t}] if authentications.blank?
+
+    authentications.collect do |auth|
+      label =
+        case auth.authtype
+        when "default" then _("Default")
+        when "bearer" then _("Bearer")
```

Comment:
@chessbyte I think it should work...

```ruby
_(auth.authtype.capitalize)
```

Now... even if it works I am not a big fan of that. I am more in favor of a table (even though the table flexibility is not strictly needed at the moment)

```ruby
AUTH_TYPE_LABEL_MAP = {
  "default" => "Default",
  ...
}
...
_(AUTH_TYPE_LABEL_MAP[auth.authtype.capitalize] || "Unknown")
```